tests/unit_tests/Commands/Self/ConsoleCommandTest.php

```diff
@@ -28,6 +30,30 @@ public function setUp()
      */
     public function testConsole()
     {
+        if (version_compare(PHP_VERSION, '7.1.0') < 0) {
+            $out = $this->command->console('site.env');
+            $this->assertNull($out);
+        }
+    }
+
+    public function testConsoleIncompatibleMessage()
+    {
+        $config = $this->getMockBuilder(TerminusConfig::class)
```

Comment:
Suggestion: Suggestion: use `$this->markTestIncomplete("Incompatible message only printed on php >= 7.1.");` when PHP < 7.1.0. Then remove all of the $config `test_mode_pass` stuff and associated mocks.